src/include/sof/audio/igo_nr/igo_nr_comp.h

```diff
@@ -35,11 +35,6 @@ struct comp_data {
 	uint32_t source_rate;	/* Sample rate in Hz */
 	uint32_t sink_format;	/* For used PCM sample format */
 	uint32_t source_format;	/* For used PCM sample format */
-	int32_t source_frames;	/* Nominal # of frames to process at source */
-	int32_t sink_frames;	/* Nominal # of frames to process at sink */
-	int32_t source_frames_max;	/* Max # of frames to process at source */
-	int32_t sink_frames_max;	/* Max # of frames to process at sink */
-	int32_t frames;		/* IO buffer length */
```

Comment:
aha, this explains that change in an earlier commit now...

Reply:
Yep!

Reply:
but can this be merged into that commit - the first one in the series?